scripts/hid_configurator/configurator_cli.py

```diff
@@ -0,0 +1,142 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+import argparse
+import logging
+
+from configurator_core import DEVICE
+from configurator_core import get_device_pid, open_device, fwinfo, fwreboot, change_config, fetch_config, dfu_transfer
+
+
+def progress_bar(permil):
+    length = 40
+    done_len = length * permil // 1000
+    progress_line = '[' + '*' * done_len + '-' * (length - done_len) + ']'
+    percent = permil / 10.0
+    print('\r{} {}%'.format(progress_line, percent), end='')
+
+
+def perform_dfu(dev, args):
+    dfu_image = args.dfu_image
+    recipient = get_device_pid(args.device_type)
+
+    success = dfu_transfer(dev, recipient, dfu_image, progress_bar)
+
+    if success:
+        print('DFU transfer completed')
+        return True
```

Comment:
there seems to be no need for returning any value as it is not used

Reply:
done